src/v2/es-utils.js

```diff
@@ -45,28 +45,47 @@ class ESClass {
     this.client = new elasticsearch.Client(esConfig);
   }
 
-  async ping(timeout) {
-    await this.client.ping({ requestTimeout: (timeout || 5000) })
-      .then(function (resp) {
-        return resp;
-      }, function (err) {
-        throw (err);
-      });
+  async healthy(timeoutSec = 90, status = 'green') {
```

Comment:
I still have dififculty with these names (eg: "what does `healthy` do/mean?" I can't clean any information from the name.)

Reply:
For most of the functions in this class, i started by matching the names of those offered by the elasticsearch.js library. One idea is something like `getEsHealth`, `getEsInfo`, `getEsPort`, `getEsTemplate`, `setKbDefaultIndex`, etc. - would that be better? I thought about this before but didn't want to make too many changes in this one PR ...